src/ApiMockerController.php

```diff
@@ -108,7 +108,13 @@ protected function getNewResponse()
 
         $status = array_get($this->config, 'status', 200);
 
-        return new Response($content, $status, ['Content-Type' => 'application/json']);
+        $header = ['Content-Type' => 'application/json'];
+
+        if (! empty($this->config['header'])) {
```

Comment:
perhaps use `$headers` and `$this->config['headers']` instead of `$header` and `$this->config['header']` since there can be multiple headers